collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py

```diff
@@ -754,6 +826,57 @@ def generate_pgroups_dict(array):
     return pgroups_info
 
 
+def generate_rl_dict(array):
+    rl_info = {}
+    api_version = array._list_available_rest_versions()
+    if ACTIVE_DR_API in api_version:
+        rlinks = array.list_pod_replica_links()
```

Comment:
This line returns an error if the `purestorage` sdk version is lower than 1.19. After an upgrade to 1.19 it's fine.

Reply:
Added warning message